R/foverlaps.R

```diff
@@ -104,10 +105,10 @@ foverlaps <- function(x, y, by.x = if (!is.null(key(x))) key(x) else key(y), by.
               end = yintervals[2L], any =,
               within =, equal = yintervals)
   call = construct(head(ynames, -2L), uycols, type)
-  if (verbose) {last.started.at=proc.time()[3];cat("unique() + setkey() operations done in ...");flush.console()}
+  if (verbose) {last.started.at=proc.time();cat("unique() + setkey() operations done in ...");flush.console()}
   uy = unique(y[, eval(call)])
-  setkey(uy)[, `:=`(lookup = list(list(integer(0))), type_lookup = list(list(integer(0))), count=0L, type_count=0L)]
-  if (verbose) {cat(round(proc.time()[3]-last.started.at,3),"secs\n");flush.console}
+  setkey(uy)[, `:=`(lookup = list(list(integer(0L))), type_lookup = list(list(integer(0L))), count=0L, type_count=0L)]
+  if (verbose) {cat(timetaken(last.started.at)); flush.console()}
```

Comment:
I'm a bit confused as to how these changes are related to this PR (1 -> 1L like enhancements). But I'm guessing you've discussed these changes with Matt.

Reply:
@arunsrinivasan the Codecov report was failing pretty miserably on the PR, since it seems to have uncovered a lot of un-tested lines (this one in particular would have come up from `3` becoming `3L`, for example